Sell/FulfilmentPoint/Tests/FulfilmentPointRepositoryTests.swift

```diff
@@ -0,0 +1,332 @@
+//
+//  FulfilmentPointRepositoryTests.swift
+//  RealifeTechTests
+//
+//  Created by Mickey Lee on 22/06/2021.
+//  Copyright © 2021 Realife Tech. All rights reserved.
+//
+
+import XCTest
+import Apollo
+@testable import RealifeTech
+
+final class FulfilmentPointRepositoryTests: XCTestCase {
+
+    // MARK: - Helper
+
+    private func makeGraphQLManagerAndSUT<DataType>(
+        ofType type: DataType.Type
+    ) -> (graphQLManager: MockGraphQLManager<DataType>, sut: FulfilmentPointRepository) {
+        let graphQLManager = MockGraphQLManager<DataType>()
+        let sut = FulfilmentPointRepository(graphQLManager: graphQLManager)
+        return (graphQLManager: graphQLManager, sut: sut)
+    }
+
+    private func makeFulfilmentPointFragment(id: String = "1") ->  ApolloType.FragmentFulfilmentPoint {
+        ApolloType.FragmentFulfilmentPoint(id: id)
+    }
+
+    private func makeFulfilmentPointCategoryFragment(id: String = "1") -> ApolloType.FragmentFulfilmentPointCategory {
+        ApolloType.FragmentFulfilmentPointCategory(id: id)
+    }
+}
+
+// MARK: - FulfilmentPoint queries tests
+extension FulfilmentPointRepositoryTests {
+
+    func test_getFulfilmentPoints_graphQLManagerHasData_completeWithSuccessCase() {
+        let tuple = makeGraphQLManagerAndSUT(ofType: ApolloType.GetFulfilmentPointsQuery.Data.self)
```

Comment:
I believe this could be written as `let (graphQLManager, sut) = makeGraphQLManagerAndSUT(ofType: ApolloType.GetFulfilmentPointsQuery.Data.self)`

- [ ] Consider using tuple decomposition (& for other tests)